dagger/blockexchange/engine.nim

```diff
@@ -112,62 +125,126 @@ proc stop*(b: BlockExcEngine) {.async.} =
 
   trace "NetworkStore stopped"
 
+proc discoverOnDht(b: BlockExcEngine, bd: BlockDiscovery) {.async.} =
+  bd.lastDhtQuery = Moment.fromNow(10.hours)
+  defer: bd.lastDhtQuery = Moment.now()
+
+  let
+    blockId = bd.toDiscover.toNodeId()
```

Comment:
Maybe something like to `toHandle` or `toHash` is more appropriate here, `toNodeId` is a bit confusing :)